randovania/data/json_data/prime1.txt

```diff
@@ -1580,411 +1583,799 @@ Tallon Overworld
+      Trivial
 
 ----------------
 Connection Elevator to Deck Beta
 Asset id: 3832686831
-> Dock to Deck Beta Conduit Hall; Heals? False
+> Dock to Deck Beta Conduit Hall; Heals? False; Spawn Point
   * Normal Door to Deck Beta Conduit Hall/Dock to Connection Elevator to Deck Beta
+  > Dock to Hydro Access Tunnel
+      Space Jump Boots and Gravity Suit
 
 > Dock to Hydro Access Tunnel; Heals? False
   * Normal Door to Hydro Access Tunnel/Dock to Connection Elevator to Deck Beta
+  > Dock to Deck Beta Conduit Hall
+      Space Jump Boots and Gravity Suit
 
 ----------------
 Hydro Access Tunnel
 Asset id: 4290029926
-> Dock to Great Tree Hall; Heals? False
+> Dock to Great Tree Hall; Heals? False; Spawn Point
   * Normal Door to Great Tree Hall/Dock to Hydro Access Tunnel
+  > Dock to Connection Elevator to Deck Beta
+      Morph Ball Bomb and Morph Ball and Gravity Suit
 
 > Dock to Connection Elevator to Deck Beta; Heals? False
   * Normal Door to Connection Elevator to Deck Beta/Dock to Hydro Access Tunnel
+  > Dock to Great Tree Hall
+      Morph Ball Bomb and Morph Ball and Gravity Suit
+  > Pickup (Energy Tank)
+      Morph Ball Bomb and Morph Ball and Gravity Suit
+
+> Pickup (Energy Tank); Heals? False
+  * Pickup 68
+  > Dock to Connection Elevator to Deck Beta
+      Morph Ball Bomb and Morph Ball and Gravity Suit
 
 ----------------
 Great Tree Hall
 Asset id: 4101881435
-> Dock to Hydro Access Tunnel; Heals? False
+> Dock to Hydro Access Tunnel; Heals? False; Spawn Point
   * Normal Door to Hydro Access Tunnel/Dock to Great Tree Hall
+  > In front of the Gate
+      Space Jump Boots
 
 > Dock to Great Tree Chamber; Heals? False
   * Normal Door to Great Tree Chamber/Dock to Great Tree Hall
+  > In front of the Gate
+      After Great Tree Hall Bars Gate
 
 > Dock to Transport Tunnel D; Heals? False
-  * Normal Door to Transport Tunnel D/Dock to Great Tree Hall
+  * Ice Door to Transport Tunnel D/Dock to Great Tree Hall
+  > In front of the Gate
+      After Great Tree Hall Bars Gate
 
 > Dock to Life Grove Tunnel; Heals? False
-  * Normal Door to Life Grove Tunnel/Dock to Great Tree Hall
+  * Ice Door to Life Grove Tunnel/Dock to Great Tree Hall
+  > In front of the Gate
+      After Great Tree Hall Bars Gate
 
 > Dock to Transport Tunnel E; Heals? False
-  * Normal Door to Transport Tunnel E/Dock to Great Tree Hall
+  * Ice Door to Transport Tunnel E/Dock to Great Tree Hall
+  > In front of the Gate
+      Space Jump Boots
+
+> Event - Open Gate; Heals? False
+  * Event Great Tree Hall Bars Gate
+  > In front of the Gate
+      Trivial
+
+> In front of the Gate; Heals? False
+  > Dock to Hydro Access Tunnel
+      Trivial
+  > Dock to Great Tree Chamber
+      X-Ray Visor and Space Jump Boots and After Great Tree Hall Bars Gate
+  > Dock to Transport Tunnel D
+      After Great Tree Hall Bars Gate
+  > Dock to Life Grove Tunnel
+      Space Jump Boots and Morph Ball and Spider Ball and After Great Tree Hall Bars Gate
```

Comment:
Move these connections to Dock to Transport Tunnel D, and remove the gate events from them.